src/applications/mhv-medical-records/actions/refresh.js

```diff
@@ -3,5 +3,20 @@ import { getRefreshStatus } from '../api/MrApi';
 
 export const fetchRefreshStatus = () => async dispatch => {
   const response = await getRefreshStatus();
+
+  const statusList = response?.facilityExtractStatusList || [];
+  const mostRecentLastRequested =
+    statusList.length > 0
+      ? Math.max(...statusList.map(extract => extract.lastRequested ?? 0))
+      : 0; // Fallback to 0 (beginning of time) if no valid data
+
+  window.localStorage.setItem('lastPhrRefreshDate', mostRecentLastRequested);
   dispatch({ type: Actions.Refresh.GET_STATUS, payload: response });
 };
+
+export const setStatusPollBeginDate = statusPollBeginDate => async dispatch => {
```

Comment:
Is the async in this function necessary?

Reply:
Nope!